common/src/main/java/com/datastrato/gravitino/dto/rel/ColumnDTO.java

```diff
@@ -228,17 +229,13 @@ public ColumnDTO build() {
    *     non-nullable with a null default value, this method will throw an IllegalArgumentException.
    */
   public void validate() throws IllegalArgumentException {
-    if (name() == null || name().isEmpty()) {
-      throw new IllegalArgumentException("Column name cannot be null or empty.");
-    }
-    if (dataType() == null) {
-      throw new IllegalArgumentException("Column data type cannot be null.");
-    }
-    if (!nullable()
-        && defaultValue() instanceof LiteralDTO
-        && ((LiteralDTO) defaultValue()).dataType().equals(Types.NullType.get())) {
-      throw new IllegalArgumentException(
-          "Column cannot be non-nullable with a null default value: " + name());
-    }
+    Preconditions.checkArgument(
+        !Strings.isNullOrEmpty(name()), "Column name cannot be null or empty");
```

Comment:
`StringUtils.isNotEmpty()` may be better. This makes it difficult for users to comprehend `!`.